webhooks/README.md

```diff
@@ -1,69 +1,70 @@
 # Webhooks
 
-Fidel uses webhooks to notify your application when relevant events happen in your account. This functionality can be used to receive data from events not triggered by direct API requests or to receive the data in a service that is not responsible for making the API request but needs to consume the response.
+The Fidel API uses [webhooks](https://en.wikipedia.org/wiki/Webhook) to notify your application when relevant events happen in your account. You can use this functionality to receive data from events not triggered by direct API requests. You can also use this functionality to receive the data in a service that is not responsible for making the API request but needs to consume the response.
 
-Fidel will notify your registered webhook URLs as the event happens. For example, when a customer makes a payment with a linked Mastercard on a participating location, a `transaction.auth` event is sent in real-time to the specified webhook URL with the transaction object in the request payload.
+There are currently nine webhooks available to use with the Fidel API, each corresponding to an event happening in your account: `brand.consent`, `card.linked `, `card.failed `, `program.status`, `transaction.auth`, `transaction.auth.qualified`, `transaction.clearing`, `transaction.clearing.qualified` and `transaction.refund`. Each of them requires an HTTPS URL to be registered.
```

Comment:
I would avoid counting how many there are, this is something we will forget to update at some point